addon/services/permissions.js

```diff
@@ -83,14 +83,14 @@ export default Service.extend(Evented, {
     return routeTreePermissions;
   },
 
-  startWatchingTransitions() {
+  enableRouteValidation() {
     if (this.isWatchingTransitions) {
```

Comment:
Care to rename this to `isRouteValidationEnabled`?

Reply:
I didn't think it was necessary since this inside the service code (not public api) which knows that it is watching transitions to handle everything.

But I don't mind changing it either.

Reply:
Done.

Reply:
Yeah, I just thought it would be more consistent with the method name.